tools/update_vk_wrappers.py

```diff
@@ -86,20 +87,43 @@ def gen_wrappers(methods, xml):
     content += "    return Result(\"Vulkan: Unable to "
     content += "load {} pointer\");\n".format(method)
     content += "  }\n"
-    content += "  ptrs_.{} = [ptr](".format(method)
+
+    # if delegate is not null ...
+    content += "  if (delegate != nullptr) {\n"
```

Comment:
What if we make this:
```
if (delegate && delegate->LogGraphicsCalls()) {
```

Add that second method to the delegate, then we add that second method to the delegate and always set it in the sample app? We're possibly going to want delegate for things like loading files so we don't want to just use the existence here. Having the method saves plumbing the flag all the way here.